packages/react-server/core/util/PageUtil.js

```diff
@@ -183,11 +183,11 @@ function standardizeDebugComments(debugComments) {
 }
 
 function standardizeMetaTags(metaTags) {
-	return PageUtil.makeArray(metaTags).map(metaTag => Q(metaTag));
+	return PageUtil.makeArray(metaTags).map(metaTag => Promise.resolve(metaTag));
```

Comment:
The binding of `this` within the `resolve` method.

In practice, I expect `Promise.resolve` doesn't use its context so `.map(Promise.resolve)` would be fine.  I'd have to read `Promise.resolve` to be sure, though.

Definitely looks way cleaner that way!